File: 8_kyu/Find_Multiples_of_a_Number.lua
local solution = {}

-- Generate table with number step, 2*step, 3*step, ... <= limit.
function solution.find_multiples(step, limit)
  local result = {}
  for i = step, limit, step do
    table.insert(result,i)
  end
  return result
end

return solution

File: 8_kyu/Find_Multiples_of_a_Number.cs
using System.Collections.Generic;
using System.Linq;

public class Kata
{
  // Returns list with number step, 2*step, 3*step, ... <= limit.
  public static List<int> FindMultiples(int step, int limit) =>
    Enumerable.Range(1,limit/step).Select( i => i*step ).ToList();
}
